plugins/services/src/js/containers/service-connection/ServiceConnectionEndpointList.tsx

```diff
@@ -44,8 +43,8 @@ class ServiceConnectionEndpointList extends React.Component {
     return getDisplayValue(protocol);
   }
 
-  getHostPortValue(portDefinition, service) {
-    if (!service.requirePorts) {
+  getHostPortValue(portDefinition) {
```

Comment:
there also is a `getHostPortValue` in `ServicePodConnectionEndpointList`. do we need to adapt that as well?

Reply:
Fixed it

Reply:
nice! 👍